packages/tools/bcm2835-bootloader/files/3rdparty/bootloader/config.txt

```diff
@@ -24,55 +22,26 @@
 # Memory (System/GPU configuration )
 ################################################################################
 
-# default GPU memory split (do not change if you do not know what you are doing)
+# Default GPU memory split (do not change if you do not know what you are doing)
   gpu_mem=128
 
-# SDRAM size based GPU memory split, uncomment and setup to override 'gpu_mem'
+# Configure GPU memory based on SDRAM size - overrides above setting
   gpu_mem_256=112
   gpu_mem_512=160
   gpu_mem_1024=256
 
 ################################################################################
-# Overclocking settings
-# WARNING: Do not change/enable if you do not know what you are doing!
-#          The System may become unstable or you can have data corruption or
-#          you can loose your warranty if you set wrong settings
-#
-# please read: http://elinux.org/RPi_config.txt#Overclocking_configuration
+# Various settings see also: http://elinux.org/RPi_config.txt
```

Comment:
Can you link to https://www.raspberrypi.org/documentation/configuration/config-txt.md
That is official documentation and is more likely to be correct.